build.gradle.kts

```diff
@@ -57,6 +59,7 @@ dependencies {
     testImplementation("org.testcontainers:mysql")
     testImplementation("com.github.dasniko:testcontainers-keycloak:$testContainersKeycloakVersion")
     testImplementation("fi.metatavu.jaxrs.testbuilder:jaxrs-functional-test-builder:$jaxrsFunctionalTestBuilderVersion")
+    testImplementation("io.netty:netty-resolver-dns-native-macos")
```

Comment:
not necessary since there's same "implementation" dependency. But are these 2 needed at all?